extras/bmp280/bmp280.h

```diff
@@ -93,7 +94,8 @@ typedef enum {
 typedef struct {
     BMP280_Mode mode;
     BMP280_Filter filter;
-    BMP280_Oversampling oversampling;  // pressure oversampling
+    BMP280_Oversampling oversampling_pressure;
+    BMP280_Oversampling oversampling_temperature;
```

Comment:
If people want for example, read faster the pressure, they can disable other measurement like temperature. 
Lets open temperature oversampling for more custom choice. User will probably do some tests if professional usage.
Maybe we can preset an optimized selection ? or comment the tab from datasheet.